keyboards/ai/readme.md

```diff
@@ -0,0 +1,27 @@
+# ai
+
+![ai](https://i.imgur.com/4l6BzRh.png)
+
+_This is a small open source 5x4-1 macropad powered by QMK_
+
+-   Keyboard Maintainer: [Louis Dalibard](https://github.com/make42)
+-   Hardware Supported: _The PCB for ai is located here: [ai](https://github.com/make42/ai)_
+-   Hardware Availability: _Links to where you can find this hardware_
```

Comment:
```suggestion
This is a small open source 5x4-1 macropad powered by QMK

* Keyboard Maintainer: [Louis Dalibard](https://github.com/make42)
* Hardware Supported: The PCB for ai is located here: [ai](https://github.com/make42/ai)
* Hardware Availability: Links to where you can find this hardware
```
Last point still needs to be filled out.

Reply:
I'm going to add links for the parts in a second commit